src/group.cpp

```diff
@@ -1,22 +1,32 @@
 #include "group.h"
+#include <mutex>
```

Comment:
`group.h` already includes `mutex`, should be removed

Reply:
Fixed by accident during sync merge (a28765a).